buildkite/schedule_and_publish/get_commits.py

```diff
@@ -1,35 +1,27 @@
-import re
-
 from config import Config
 from models.benchmarkable import Benchmarkable
 from integrations.github import Github
 from logger import log
 
-# apache/arrow commits into master branch have messages
-# that contain originating pull request number in phrases like "\n\nCloses #10973 from ..."
-pattern = "\nCloses #(.*) from"
-compiled_pattern = re.compile(pattern)
-
 
-def get_originating_pull_number(commit_dict):
+def get_originating_pull_number(github, commit_dict):
     try:
-        return (
-            compiled_pattern.search(commit_dict["commit"]["message"]).group(1).strip()
-        )
+        return github.get_pulls(commit_dict["sha"])[0]["number"]
     except Exception as e:
         log.error(
             f"Unable to get originating pull number for commit = {commit_dict['sha']} "
-            f"from message = {commit_dict['commit']['message']}"
+            f"with message = {commit_dict['commit']['message']}"
```

Comment:
We can keep it in cause more info is better than less, but we don't _really_ need the message here anymore now that we get the PR number from git(hub) data, right?

Reply:
We don't need it but it helps for debugging.